features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS IDN_BASE_TABLE (
             PRODUCT_NAME VARCHAR(20),
             PRIMARY KEY (PRODUCT_NAME)
-)ENGINE INNODB;
+)DEFAULT CHARACTER SET latin1 ENGINE INNODB;
```

Comment:
As this is a known limitation for MySQL 8 and in the [documentation](https://is.docs.wso2.com/en/6.0.0/deploy/environment-compatibility/#tested-dbmss) itself, explicitly mentioned that, databases needs to be created with `latin1`. Hence IMO, we don't need to add this in the dbscripts @DMHP WDYT?

BTW, Can you check whether this syntax errors are occurred in bit lower version other than the MYSQL8?

Reply:
As this is a known limitation for MySQL 8 and in the [documentation](https://is.docs.wso2.com/en/6.0.0/deploy/environment-compatibility/#tested-dbmss) itself, explicitly mentioned that, databases needs to be created with `latin1`. Hence IMO, we don't need to add this in the dbscripts @DMHP WDYT?

Reply:
I have tested this using 5.7.12 MySQL Community Server (GPL) and it did not produce any errors.